src/frontend/src/pages/DRCPage/FillerCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Card, CardContent, Grid, Typography, useTheme } from '@mui/material';
+
+interface FillerCardProps {
+  day: Number;
+}
+
+const FillerCard: React.FC<FillerCardProps> = ({ day }) => {
+  const FillerCardTitle = () => (
+    <Grid container alignItems="center" margin={0} padding={0}>
+      <Grid item></Grid>
+      <Grid item xs display="flex" justifyContent="flex-end">
+        <Typography variant="h6" marginRight={1} color="grey">
+          {day}
+        </Typography>
+      </Grid>
+    </Grid>
```

Comment:
you should be able to do all this with one Box component wrapping the Typography, and then just move it to where this component is called below to simplify this code